teku/src/main/java/tech/pegasys/teku/cli/options/BeaconRestApiOptions.java

```diff
@@ -76,6 +79,22 @@
       hidden = true)
   private int maxPendingEvents = DEFAULT_MAX_EVENT_QUEUE_SIZE;
 
+  @Option(
+      names = {"--Xrest-api-max-url-length"},
+      paramLabel = "<INTEGER>",
+      hidden = true)
+  public void setMaxUrlLength(int maxUrlLength) {
```

Comment:
huh, I don't think I've ever seen us use setters for options. It's a great pattern and probably a bunch of other places we could take advantage of it.  Only downside is I'm guessing it doesn't show the default value in help when done this way.

Probably should add a description in the annotation.